pybio/spec/utils.py

```diff
@@ -96,6 +97,11 @@ def get_instance(node: Union[nodes.SpecWithKwargs, nodes.WithImportableSource],
 
         return get_instance(node.spec, **joined_spec_kwargs)
     elif isinstance(node, nodes.WithImportableSource):
+        if not isinstance(node.source, ImportedSource):
+            raise PyBioTransformationOrderException(
```

Comment:
I think there is no need for the custom exception type here. We only need custom exceptions when user can catch them explicitly to take some action, but in this case it looks like ValueError with same error message is a better fit.

Reply:
Alright, we might want to emphasize in the module api that load_spec_with_kwargs and get_instance are the appropriate funcitons to use... they are a bit hidden away in spec.utils atm